client/my-sites/themes/themes-selection.jsx

```diff
@@ -34,6 +31,35 @@ const ThemesSelection = React.createClass( {
 		tier: React.PropTypes.string,
 	},
 
+	getDefaultProps() {
+		return { search: '' };
+	},
+
+	doSearch( searchString ) {
+		const filterRegex = /\w+\:\s*([\w-]+)/g;
```

Comment:
I was wondering if we were really allowing numeric parts of a filter value, let alone key, or only alphabetic?